sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py

```diff
@@ -213,6 +213,7 @@ def __init__(self, codes):
             ),
             NumberRegField(consts.CTRLS_ADVT_FIELD, self.getaddr(0x1, 155),
                 RegBitField(consts.TX_DISABLE_SUPPORT_FIELD, 1),
+                RegBitField(consts.RX_DISABLE_SUPPORT_FIELD, 2),
```

Comment:
@vdahiya12 this should be accessing byte 156 and not byte 155.

![image](https://github.com/user-attachments/assets/183d6c5d-03b1-42c6-8fff-d8cdbbc8671d)

Reply:
fixed